src/main/java/uk/gov/hmcts/reform/jobscheduler/model/HttpAction.java

```diff
@@ -1,15 +1,22 @@
 package uk.gov.hmcts.reform.jobscheduler.model;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import org.hibernate.validator.constraints.NotBlank;
 import org.springframework.http.HttpMethod;
 
 import java.util.Map;
+import javax.validation.constraints.NotNull;
 
 public class HttpAction {
 
+    @NotBlank
```

Comment:
Maybe we could require a valid URL here?

Reply:
done